main.go

```diff
@@ -1,73 +1,95 @@
+/*
```

Comment:
why we need to move the main to root folder?

Reply:
Expected by kubebuilder/controller-gen and now operator-sdk.

Reply:
this is quite weird. No option to use a custom location?

Reply:
None other than manual move again to the same place. But this is about:
- standardization: all operators will have the main on the root
- code generation: every time we advance the API (or do other stuff) the main may be updated. If is not in the expected place it  will fail.
Since we don't have any special code in main I don't think it matters so much.

Reply:
really? what a weird decision from them. I'll dig into their doc to understand their rationale.